zsh-proxy.plugin.zsh

```diff
@@ -34,20 +34,30 @@ __check_ip() {
 	echo "========================================"
 	echo "Check what your IP is"
 	echo "----------------------------------------"
-	echo -n "IPv4: "
-	curl -s -k https://api-ipv4.ip.sb/ip
+	ipv4=$(curl -s -k https://api-ipv4.ip.sb/ip)
+	if [ "$ipv4" = "" ]
```

Comment:
Avoid using `[ ]` which will spawn a child_process of `test`, causing performance issue. Use `[[ ]]` instead.

Reply:
@mouyase plz use https://www.shellcheck.net and https://github.com/tmknom/shfmt